### ISSUE TITLE
Latest jinja2 release on pypi breaks pygsti

### ISSUE THREAD

**Issue by peendebak** [pyGSTio/pyGSTi#239]
**Describe the bug**

The method `evalcontextfilter` was replaced in `jinja2` 
(changes are described here: https://github.com/pallets/jinja/blob/6f79daafe868b8ef20baecdcb45d12299529ed2e/CHANGES.rst#version-310)

**To Reproduce**
1. Install `pygsti` with `jinja2` release from pypi
2. Run GST example

Error generated:
```
...
File /usr/local/lib/python3.9/site-packages/pygsti/report/merge_helpers.py:469, in _make_jinja_env(static_path, template_dir, render_options, link_to)
    465     contents = offline_loader.get_source(env, filename)[0]
    466     return jinja2.Markup(contents)
    468 @jinja_filter
--> 469 @jinja2.evalcontextfilter
    470 def render(eval_ctx, value):
    471     html = _render_as_html(value, render_options or {}, link_to)
    472     return jinja2.Markup(html) if eval_ctx.autoescape else html
AttributeError: module 'jinja2' has no attribute 'evalcontextfilter'
AttributeError: module 'jinja2' has no attribute 'evalcontextfilter'
```

**Expected behavior**

No error should be generated

**Environment (please complete the following information):**
 - pyGSTi version [e.g. v0.9.7.4]: 0.9.10
 - python version [e.g. 3.7, 2.7]: 3.9
 - OS [e.g. OSX 10.14, Ubuntu 16.04 LTS]: Windows

**Additional context**

PS: If external PRs are excepted, I can fix this and make a PR.

@enielse

**Comment by coreyostrove** [comment]
Hi @peendebak, 

Thank you for reporting this issue and apologies for taking half a year to get around to fixing this (admittedly if I hadn't run into this issue earlier today it probably would have still been a while). This is fixed on the branch "bugfix-jinja" and will be merged into the branch "develop" soon. Whenever we make a new release this will also be integrated into "master." I'm closing this for now, but if you run into any related issued please feel free to reopen this.

Thanks again,

Corey

**Comment by sserita** [comment]
Closed as this is now in 0.9.11 release
